Packs/TeamViewer/Integrations/TeamViewerEventCollector/TeamViewerEventCollector_description.md

```diff
@@ -0,0 +1,8 @@
+## Team Viewer
+Use this integration to collect events automatically from Team Viewer.
+You can also use the ***teamviewer-get-events*** command to manually collect events.
+
+### Script Token
+[For instructions on how to generate your script token, click here](https://community.teamviewer.com/English/kb/articles/109647-use-the-teamviewer-api).
```

Comment:
Please add here the steps to create the script token and then add the link (for more information....)